docs/user_guide/secure_payloads.rst

```diff
@@ -0,0 +1,93 @@
+Secure Payloads
+================
+
+.. warning::
+    This page is still under development and not complete. It will be so until
+    this warning is removed.
+
+Trusted payloads offers the ability to securely provision secrets to an enrolled node.
```

Comment:
I'd have written "Trusted payloads offer" unless we implicitly refer to the functionality of "Trusted payloads"